deploy/helm/kuberhealthy/templates/configmap.yaml

```diff
@@ -30,4 +30,18 @@ data:
           description: Kuberhealthy shows that the cluster is not healthy
 {{- end -}}
 {{- end -}}
-
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+    name: special-config
```

Comment:
the name of the configmap should not be `special-config`.  Let's make it simply `kuberhealthy`.  The type will be obvious.

Reply:
Changes made with 6bf8b54